python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -668,6 +715,58 @@ async def park_dome(self) -> None:
                 check_in_position=True,
             )
 
+    async def unpark_dome(self) -> None:
+        """Un-Park the dome by moving it a small delta amount."""
+
+        await self.assert_all_enabled(
+            message="All components need to be enabled for un-parking the Dome."
+        )
+
+        self.log.info("Checking if the dome is currently PARKED.")
+        az_motion = await self.rem.mtdome.evt_azMotion.aget(timeout=self.fast_timeout)
+
+        if az_motion.state == MTDome.MotionState.PARKED:
+            self.log.info("Dome is currently PARKED. Proceeding to un-park.")
+
+            current_position = await self.rem.mtdome.tel_azimuth.aget(
+                timeout=self.fast_timeout
+            )
+
+            az_delta = 0.1  # A small move to un-park the Dome
+            unparked_position = current_position.positionActual + az_delta
+            unparked_velocity = 0.0  # The Dome defaults to a `crawl` velocity
+
+            self.rem.mtdome.evt_azEnabled.flush()
+            self.rem.mtdome.evt_azMotion.flush()
+
+            await self.rem.mtdome.cmd_moveAz.set_start(
+                position=unparked_position,
+                velocity=unparked_velocity,
+                timeout=self.park_dome_timeout,
+            )
+
+            # The cmd_MoveAz should issue a azEnabled event set to ENABLED
+            await self.wait_for_dome_azEnabled(timeout=self.park_dome_timeout)
```

Comment:
So you really need to look for this event? Why  not rely on the `wait_for_dome_state`?

Reply:
on the azEnabled event, I'm following the advice given by you and Wouter on the issue: https://rubinobs.atlassian.net/browse/DM-45608?focusedCommentId=523258 specifically:

`Also check the azMotion, and azEnabled events to verify UNPARKED status.`


but happy to do otherwise.

Reply:
as per @tribeiro comment: 
`velocity will always be zero, no matter what.. you can even ignore that parameter and don't set it as the default is zero.`